package.json

```diff
@@ -2,32 +2,89 @@
   "name": "serve",
   "version": "13.0.4",
   "description": "Static file serving and directory listing",
-  "scripts": {
-    "test": ""
-  },
-  "files": [
-    "bin"
-  ],
-  "repository": "vercel/serve",
-  "bin": {
-    "serve": "./bin/serve.js"
-  },
   "keywords": [
     "vercel",
     "serve",
     "micro",
     "http-server"
   ],
+  "repository": "vercel/serve",
   "license": "MIT",
+  "type": "module",
+  "bin": {
+    "serve": "./build/main.js"
+  },
+  "files": [
+    "build/",
+    "package.json",
+    "readme.md",
+    "license.md",
+    ".npmrc"
```

Comment:
I don't think `.npmrc` should be published

Reply:
Oh that's cool - I didn't know that, will revert.

Reply:
No problem! That's what code reviews are all about ;)